pyproject.toml

```diff
@@ -16,4 +62,4 @@ exclude = '''
     | dist
   )/
 )
-'''
+'''
```

Comment:
Newline. navargus lacks precommit?

Reply:
No, it has it, I just forgot to install it